Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1884,7 +1884,27 @@ UpdateOsBootMediumInfo (
   OUT  OS_BOOT_OPTION_LIST  *OsBootOptionList
 )
 {
-  FillBootOptionListFromCfgData (OsBootOptionList);
+  OS_BOOT_OPTION             *BootOptionCfgData;
+  UINT32                      Idx;
+
+  for (Idx = 0; Idx < MAX_BOOT_OPTION_CFGDATA_ENTRY; Idx++) {
+    BootOptionCfgData = (OS_BOOT_OPTION *)FindConfigDataByTag (CDATA_BOOT_OPTION_TAG + Idx);
+    if (BootOptionCfgData->DevType == 0x5){
+      if (FeaturePcdGet (PcdPreOsCheckerEnabled) && PchIsSciSupported ()) {;

```

Comment:
extra semicolon at the end line need to remove

Reply:
removed, thanks.